vault-ui/src/pages/dashboard/UserSettingsPage.tsx

```diff
@@ -160,6 +170,42 @@ function RegisterPasskeyButton() {
   );
 }
 
+function DeletePasskeyButton({ passkeyId }: { passkeyId: string }) {
+  const { mutateAsync: deletePasskeyAsync } = useMutation(deleteMyPasskey);
+  const { refetch: refetchListMyPasskeys } = useQuery(listMyPasskeys);
+
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  async function handleDeletePasskey() {
+    await deletePasskeyAsync({ id: passkeyId });
+    await refetchListMyPasskeys();
+    toast.success("Passkey deleted");
+  }
+
+  return (
+    <AlertDialog open={dialogOpen} onOpenChange={setDialogOpen}>
+      <AlertDialogTrigger asChild>
+        <Button variant="outline">Delete</Button>
+      </AlertDialogTrigger>
+      <AlertDialogContent>
+        <AlertDialogHeader>
+          <AlertDialogTitle>Are you sure?</AlertDialogTitle>
+          <p>
```

Comment:
don't use `p`, there's a component for this in alert-dialog. Something "description"?

Reply:
Updated